src/ngx_stream_upsync_module.c

```diff
@@ -1155,7 +1155,7 @@ ngx_stream_upsync_consul_parse_json(void *data)
         cJSON *temp1 = cJSON_GetObjectItem(server_next, "Key");
         if (temp1 != NULL && temp1->valuestring != NULL) {
             p = (u_char *)ngx_strrchr(temp1->valuestring, '/');
-            if (ngx_stream_upsync_check_key(p) != NGX_OK) {
```

Comment:
In that case, can we move this check to ngx_stream_upsync_check_key() as well? That's where other loggers are and check/validations are done.